clang/include/clang/StaticAnalyzer/Core/PathSensitive/SMTConv.h

```diff
@@ -570,23 +572,35 @@ class SMTConv {
   // TODO: Refactor to put elsewhere
   static inline QualType getAPSIntType(ASTContext &Ctx,
                                        const llvm::APSInt &Int) {
-    return Ctx.getIntTypeForBitwidth(Int.getBitWidth(), Int.isSigned());
+    QualType Ty;
+    if (!(Ty = Ctx.getIntTypeForBitwidth(Int.getBitWidth(), Int.isSigned()))
+             .isNull())
+      return Ty;
+    // If Ty is Null, could be because the original type was a _BitInt.
+    // Get the bit size and round up to next power of 2, max char size
```

Comment:
```suggestion
    // Get the size of the _BitInt type (expressed in bits) and round it up to
    // the next power of 2 that is at least the bit size of 'char' (usually 8).
```
For a moment I was very confused by this comment, because I thought that "max char size" means that for some reason the value is limited _from above_ by 256 (the "char size" in a different sense). I tried to clarify the comment in a suggestion – but feel free to tweak it if you want.